auto_editor/BaseRule.py

```diff
@@ -37,6 +37,8 @@ def initiate_run(self, project: StructuredProjectSource):
 
                 # index of line id is taken at each iteration,
                 # as it may change if the project is modified in previous iterations
+                if file_path == 'shortcuts.dp.hpp':
+                    print('yo')
```

Comment:
why print ‘yo’ there？

Reply:
haha oh, that was left over from debugging